README.md

```diff
@@ -121,20 +121,27 @@ Written materials are provided under the [CC-BY-4.0 SA license](LICENSE.md).
 
 Please see [`AUTHORS.md`](AUTHORS.md) for a list of contributors to the workshop materials.
 
-We would like to give a special thanks to our partner for this workshop, the [Molecular Sciences Software Institute](https://molssi.org/) (MolSSI).
+We would like to give a special thanks to our partner and primary sponsor for this workshop, the [Molecular Sciences Software Institute](https://molssi.org/) (MolSSI).
```

Comment:
@armcdona would you like to change anything here?

Reply:
This looks good to me.

Reply:
thanks!